{
  "id" : "bfad32f2-ed88-4f2c-854b-c5ced3aa4c43",
  "name" : "api_2.0_sql_statements",
  "request" : {
    "url" : "/api/2.0/sql/statements/",
    "method" : "POST",
    "bodyPatterns" : [ {
      "equalToJson" : "{\"statement\":\"INSER INTO main.jdbc_test_schema.query_syntax_error_test_table (id, col1, col2) VALUES (1, 'value1', 'value2')\",\"warehouse_id\":\"dd43ee29fedd958d\",\"session_id\":\"01efe91f-12e2-1f8a-9c6a-be676187bd0f\",\"disposition\":\"EXTERNAL_LINKS\",\"format\":\"ARROW_STREAM\",\"on_wait_timeout\":\"CONTINUE\",\"wait_timeout\":\"10s\",\"parameters\":[],\"result_compression\":\"LZ4_FRAME\"}",
      "ignoreArrayOrder" : true,
      "ignoreExtraElements" : true
    } ]
  },
  "response" : {
    "status" : 200,
    "body" : "{\"statement_id\":\"01efe91f-13be-182b-8d57-8c559b73a0d3\",\"status\":{\"state\":\"FAILED\",\"error\":{\"error_code\":\"BAD_REQUEST\",\"message\":\"\\n[PARSE_SYNTAX_ERROR] Syntax error at or near 'INSER'. SQLSTATE: 42601 (line 1, pos 0)\\n\\n== SQL ==\\nINSER INTO main.jdbc_test_schema.query_syntax_error_test_table (id, col1, col2) VALUES (1, 'value1', 'value2')\\n^^^\\n\"},\"sql_state\":\"42601\"}}",
    "headers" : {
      "x-request-id" : "e9159300-64e9-43fd-a89a-ff41eb5f3bab",
      "date" : "Wed, 12 Feb 2025 08:55:15 GMT",
      "server" : "databricks",
      "x-databricks-popp-response-code-details" : "via_upstream",
      "x-databricks-shard-debug" : "oregon-staging",
      "vary" : "Accept-Encoding",
      "x-databricks-org-id" : "6051921418418893",
      "strict-transport-security" : "max-age=31536000; includeSubDomains; preload",
      "x-content-type-options" : "nosniff",
      "x-databricks-popp-routing-reason" : "deployment-name",
      "content-type" : "application/json",
      "x-databricks-apiproxy-response-code-details" : "via_upstream"
    }
  },
  "uuid" : "bfad32f2-ed88-4f2c-854b-c5ced3aa4c43",
  "insertionIndex" : 22
}